src/storage/src/source/generator.rs

```diff
@@ -44,30 +44,39 @@ enum GeneratorKind {
     Simple {
         generator: Box<dyn Generator>,
         tick_micros: Option<u64>,
+        // Load generators cannot be rendered until all of their exports are
+        // present.
```

Comment:
We chatted about this in real time during our pre-review of this PR. At that time, you said that this was unfortunate and should be fixed eventually. I will file an issue that these need to be re-designed.

> What happens if render the load gen with fewer subsources?

We don't support that. All load generator sources require `FOR ALL TABLES`. If we did support producing a subset of tables, we either would have to have not tested the scenario (somewhat unlikely) or it would have failed because we never would have produced data. Fortunately, we can see that we just [don't support rendering a subset of the subsources](https://github.com/MaterializeInc/materialize/blob/4b0f90f590dc0b0824e6a14935160a3f2253e441/src/sql/src/pure.rs#L1159-L1179).